MobilityData/gtfs-validator
# fix: Added a execution result file

**Summary:**

closes #1688

Added `execution_result.json` file containing the result of the execution of the validation.
This file should be available at the same location as the report files (report.html, report.json, etc)
Typical contents of this file:
```
{
  "status": "error",
  "error": "Validation failed"
}
``` 
or
```
{
  "status": "success",
  "error": ""
}
```

Now the endpoint always return 200.

**Testing tips**

- Deploy to staging
   - I added a manual trigger to `.github/workflows/stg_web_svc_merge.yml` but this will not be available until this branch is merged into master.
   - Meanwhile you can create your own branch and add a push to your branch trigger
   - You don't need to deploy the client.
- On the staging client web page, select a good dataset and validate it. Open the report.
- Replace `report.html` at the end of the report URL by `execution_result.json`. You should be able to load the file
- Try gain with a broken dataset

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `gradle test` to make sure you didn't break anything
- [ ] Add or update any needed [documentation](https://github.com/MobilityData/gtfs-validator/tree/master/docs) to the repo
